test/unit/test-xhr.js

```diff
@@ -751,9 +752,9 @@ describe
           .then(() => expect(sendMessageStub).to.not.have.been.called);
       });
 
-      it('should intercept if viewer untrusted but dev mode', () => {
+      it('should intercept if viewer untrusted but in local dev mode', () => {
         sandbox.stub(viewer, 'isTrustedViewer').returns(Promise.resolve(false));
-        interceptionEnabledWin.AMP_DEV_MODE = true;
+        sandbox.stub(mode, 'getMode').returns({localDev: true});
```

Comment:
You can set `test.win.AMP_MODE = {localDev: true};` to avoid the wildcard import.

Reply:
The test is actually not enclosed in the scenario test cases which mocks the window obj which is specific to testing native vs polyfill fetch, so this wouldn't work. Thanks for the input though.